### DISCUSSION TITLE
Export/Import Authors Data Functionality

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Authors#1696] [Feature Requests]
It will be good if we have EXIM functionality built in the plugin

**Comment by ojopaul** [comment]
@rizaardiyanto1412 Can elaborate more on this before it went cold, what's EXIM functionality?

**Reply by rizaardiyanto1412** [reply]
It's export import feature @ojopaul

**Reply by rizaardiyanto1412** [reply]
Related helpscout: https://secure.helpscout.net/conversation/2534822612/220701?viewId=6932143

**Reply by rizaardiyanto1412** [reply]
Currently if we want to export or import, they will need plugin like WP All Import/Export

**Reply by ojopaul** [reply]
Okay, I thought EXIM as a word mean something but reading it again now after your comment, seems you mean EX/IM(Export/Import). @rizaardiyanto1412

**Comment by ojopaul** [comment]
As a note here, we can't have Export/Import for whole plugin due to how author is tied to users and posts and by portraying full site export, it may confuse people as full export is not possible unless posts and users are exported as well to enable proper linking of authors to users and posts.

However, we can have export for the following features/screen:
- Author Boxes, Author Fields (WordPress already has this since it's a post type but if we don't mind feature duplicate, we can add it)
- Author Categories
- Settings
